### PR TITLE
Workflow Automated Test - LRQA-67555 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase [hunk 1]
```diff
@@ -30,12 +30,40 @@ definition {
...
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase:56]: Could you please change this macro to 
`JSONBlog.addEntry(
				entryContent = "Blogs Entry Content",
				entryTitle = "Blogs Entry Title");`

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
[Sent to BChan](https://github.com/CalumR23/liferay-portal/pull/10)

**Pull request closed without merging**